modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractGoCodegen.java

```diff
@@ -671,23 +664,11 @@ public ModelsMap postProcessModels(ModelsMap objs) {
                     addedOSImport = true;
                 }
             }
-
             if (this instanceof GoClientCodegen && model.isEnum) {
                 imports.add(createMapping("import", "fmt"));
             }
 
-            // if oneOf contains "time.Time" type
-            if (!addedTimeImport && model.oneOf != null && model.oneOf.contains("time.Time")) {
-                imports.add(createMapping("import", "time"));
-            }
-
             // if oneOf contains "null" type
-            if (model.oneOf != null && !model.oneOf.isEmpty() && model.oneOf.contains("nil")) {
```

Comment:
Why this was removed?

Reply:
Good catch! looks like I screwed up taking the changes from https://github.com/OpenAPITools/openapi-generator/pull/14057/files